test/scripts/jenkins_xpack.sh

```diff
@@ -34,9 +34,9 @@ installDir="$PARENT_DIR/install/kibana"
 mkdir -p "$installDir"
 tar -xzf "$linuxBuild" -C "$installDir" --strip=1
 
-
+export TEST_ES_FROM=${TEST_ES_FROM:-source}
 echo " -> Running functional and api tests"
 cd "$XPACK_DIR"
-xvfb-run node scripts/functional_tests --debug --bail --kibana-install-dir "$installDir" --esFrom=source
+xvfb-run node scripts/functional_tests --debug --bail --kibana-install-dir "$installDir" --esFrom="$TEST_ES_FROM"
```

Comment:
Instead of passing it in, what if we made the esFrom param also look at the TEST_ES_FROM environment variable? This way when we know we can install from source we can set it. It looks like there are already places which we have it: https://github.com/elastic/kibana/blob/master/packages/kbn-test/src/es/es_test_config.js#L38